datahub-web-react/src/app/ingestV2/source/IngestionSourceTableColumns.tsx

```diff
@@ -43,7 +43,23 @@ const SourceNameText = styled(Typography.Text)<{ $shouldUnderline?: boolean }>`
     font-size: 14px;
     font-weight: 600;
     color: ${colors.gray[600]};
-    line-height: normal;
+    line-height: 1.3 !important;
+    display: -webkit-box !important;
+    -webkit-line-clamp: 2 !important;
+    -webkit-box-orient: vertical !important;
+    overflow: hidden !important;
+    word-break: break-word !important;
+    white-space: normal !important;
+    text-overflow: unset !important;
+
+    &.ant-typography {
+        display: -webkit-box !important;
+        -webkit-line-clamp: 2 !important;
+        -webkit-box-orient: vertical !important;
+        overflow: hidden !important;
+        white-space: normal !important;
+    }
```

Comment:
lots of `!important` is always a general red flag for me..

just cuz ant design styles are hard to override?